src/Service.Tests/CosmosTests/QueryFilterTests.cs

```diff
@@ -1101,18 +1113,20 @@ public async Task TestQueryFilterFieldAuthWithoutSingularType()
         public async Task TestQueryFilterFieldAuth_ExcludeTakesPredecence()
         {
             string gqlQuery = @"{
-                suns(first: 1, " + QueryBuilder.FILTER_FIELD_NAME + @" : {name : {eq : ""test name""}})
+                planets(first: 1, " + QueryBuilder.FILTER_FIELD_NAME + @" : {suns: { name : {eq : ""test name""}}})
```

Comment:
please update test summary to indicate what behavior is being validated. is authZ being performed for `planets` or `suns` or both? which fields are excluded.

Reply:
Nature of the test is same. We are testing it for `sun`.... only graphql query is changed to represent that we can query `sun` through `planet` only, thats where `sun` actually resides.